tests/storage.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var request = require('supertest');
+var should = require('should');
+var assert = require('assert');
+var load = require('./fixtures/load');
+
+describe('STORAGE', function () {
+  var env = require('../env')();
+
+  before(function (done) {
+    delete env.api_secret;
+    done();
+  });
+
+  it('The storage class should be OK.', function (done) {
+    require('../lib/storage').should.be.ok;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/759804173/issues/source?bid=2449627&fileBranchId=2449709#l17)